Documentation/Components/Snackbar/snackbar.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Snackbar</title>
+    <link rel="stylesheet" href="snackbar.css">
+</head>
+
+<body>
+    <div class="snackbar flex-row">
+        <p class="para-s">Can't send picture. Retry in 5 seconds.</p>
+        <button class="snackbar-btn retry-btn">RETRY</button>
```

Comment:
Haven't used `retry-btn` class in css , if it's not necessary. Remove it

Reply:
Oh, I forgot to remove it, thank you!